lib/vault/Dockerfile

```diff
@@ -0,0 +1,45 @@
+# From https://www.bogotobogo.com/DevOps/Docker/Docker-Vault-Consul.php
+# base image
+FROM alpine:3.7
+
+# set vault version
+ENV VAULT_VERSION 1.5.0
```

Comment:
vault version should be a runtime arg instead if possible, see `lib/templates/docker-compose.yml` and `lib/templates/Dockerfile` for examples